src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough.java

```diff
@@ -487,7 +487,7 @@ void multipleCasesOnOneLine() {
         int i = 0;
         switch (i) {
         case 0: case 1: i *= i; // fall through
-        case 2: case 3: i *= i; // violation 'Fall through from prev.* br.* switch statement.'
+        case 2: case 3: i *= i;
         case 4: case 5: i *= i; // violation 'Fall through from prev.* br.* switch statement.'
```

Comment:
Why was this violation removed? I am not seeing a break or other control flow that would make this not a fall through.

Reply:
No break, but there is a comment `// fall through`

Reply:
simplified code of above 
    void multipleCasesOnOneLine() {
        int i = 0;
        switch (i) {
        case 0: 
        case 1: // ok 
            i *= i; // fall through
        case 2: // ok
        case 3: i *= i;  // no break, continue, fallthrough
        case 4:  // violation 'Fall through from prev.* br.* switch statement.'
        case 5: i *= i;        
        case 6: case 7: i *= i; // violation 'Fall through from prev.* br.* switch statement.'
            break;
        default:
            throw new RuntimeException();
        }
    }